scapy/pton_ntop.py

```diff
@@ -17,6 +17,10 @@ def inet_pton(af, addr):
     if af == socket.AF_INET:
         return socket.inet_aton(addr)
     elif af == socket.AF_INET6:
+        # Use inet_pton if available
+        if hasattr(socket, "inet_pton"):
```

Comment:
Since `socket` will, except in very rare occasions, have `inet_pton`, you might want to write:
```Python
    try:
        return socket.inet_pton(af, addr)
    except AttributeError:
        pass
```